src/common/util/game.ts

```diff
@@ -641,7 +641,12 @@ export function executeCmd(
   const [terms, definitions] = [Object.keys(vocabulary), Object.values(vocabulary)];
   const wrappedCmd = `(function (${terms.join(',')}) { return (${cmd})(); })`;
 
-  return eval(wrappedCmd)(...definitions);  // eslint-disable-line no-eval
+  try {
+    return eval(wrappedCmd)(...definitions);  // eslint-disable-line no-eval
+  } catch (error) {
+    alert(`Oops!\n\n${error}`);
```

Comment:
Hmm should we really be doing a browser alert here?

Reply:
I'll put a TODO to make this better, but it's a little tricky (we probably need an error handler in the reducer).